packages/perseus/src/__tests__/__snapshots__/renderer_test.jsx.snap

```diff
@@ -395,7 +395,7 @@ exports[`renderer snapshots correct answer: correct answer 1`] = `
                   <span
                     class="text_f1191h-o_O-LabelMedium_1rew30o-o_O-text_16ujc3c"
                   >
-                    greater/less than or equal to
+                    less than or equal to
```

Comment:
Why did this change?

Reply:
This is the correct value. The snapshot in the original PR from webapp had this wrong and I didn't notice it (the dropdown is in a closed state)